src/main/java/Library.java

```diff
@@ -566,4 +568,26 @@ public static int findLevenshteinDistance(String word1, String word2) {
       return stream.collect(Collectors.toList());
     }
   }
+
+  /**
+   * Convert InputStream to String.
+   *
+   * @param fileName file to read
+   * @throws IOException if an I/O error occurs
+   */
+  public static String inputStreamToString(String fileName) throws IOException {
+    InputStream inputStream = new FileInputStream(fileName);
+    InputStreamReader reader = new InputStreamReader(inputStream);
```

Comment:
There's a potential resource leak here. I recomment using Java's try-with-resources idiom: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Additionally we should use Java's local variable type inference, `var` syntax